packages/EDA/src/utils.ts

```diff
@@ -49,3 +49,13 @@ export function checkGeneratorSVMinputs(samplesCount: number, featuresCount: num
   if ((violatorsPercentage < PERCENTAGE_MIN) || (violatorsPercentage > PERCENTAGE_MAX))
     throw new Error(INCORERRECT_PERCENTAGE_MES);
 }
+
+// Check columns (ANOVA)
+export function checkColumns(columns: DG.ColumnList): void {
+  if (Array.from(columns.numerical).length < columns.length)
```

Comment:
Creating an array just to check whether there are non-numerical columns is expensive, rewrite it in a more efficient manner

Reply:
?